docs/package.json

```diff
@@ -38,9 +38,9 @@
     "next-sitemap": "^1.6.203",
     "prism-react-renderer": "1.2.1",
     "raw-loader": "^4.0.2",
-    "react": "latest",
+    "react": "17",
```

Comment:
should we use `^17.0.0`?

Reply:
Isn't that pretty much equivalent to `17`?

Reply:
I need to look it up 😂

Reply:
Probably wouldn't actually ever want `17.0.0` anyway. With `17` we should always get the latest version of 17, which is what we want for docs.

Reply:
Cool! Verified here https://semver.npmjs.com/ 😁